.github/workflows/azure-deploy-io-fast-login.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy (io-fast-login)
+
+on:
+  workflow_dispatch:
```

Comment:
Add `swap_slot_step` input, as defined in session-manager workflow

Reply:
not present in `function_app_deploy.yaml` re-usable workflow

Reply:
😫